### PR TITLE
Drop support for Python 3.8

### PR DESCRIPTION
**Pull request by mrtmm**

### CODE REVIEW

File: setup.py
Comment (fghaas) [inline setup.py:1]: You'll need to update the `classifiers` list as well; it still includes 3.8.

File: tutorbackup/templates/backup/build/backup/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/ubuntu:20.04
+FROM docker.io/ubuntu:22.04
```
Comment (fghaas) [inline tutorbackup/templates/backup/build/backup/Dockerfile:1]: `CHANGELOG.md` should mention that this change requires an image rebuild.

**Review by fghaas — COMMENTED** [review]
Looks good in general; I have a couple of notes.

File: tutorbackup/templates/backup/build/backup/Dockerfile (hunk 1 above)
Comment (mrtmm) [reply tutorbackup/templates/backup/build/backup/Dockerfile:1]: Ack, done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+* [Enhancement] Drop support for Python 3.8; Update the Ubuntu base image to
```
Comment (fghaas) [inline CHANGELOG.md:3]: I understand I am nitpicking, but a semicolon does not trigger capitalisation of the next word.

Thus, either use:

```suggestion
* [Enhancement] Drop support for Python 3.8. Update the Ubuntu base image to
```

... or else, use:

```suggestion
* [Enhancement] Drop support for Python 3.8; update the Ubuntu base image to
```

**Review by fghaas — COMMENTED** [review]
Just a tiny nit left. :)

**Review by fghaas — APPROVED** [review]

**Comment by fghaas** [comment]
I've dropped the Python 3.8 check from the list of required checks for this repo.

**Pull request merged**